openmc/mgxs/mgxs.py

```diff
@@ -1484,22 +1487,26 @@ def rxn_rate_tally(self):
             self._rxn_rate_tally.sparse = self.sparse
         return self._rxn_rate_tally
 
+class FissionXSBase(MGXS):
+    """A fission production multi-group cross section base class
+       for NuFission and KappaFission
+    """
 
-class FissionXS(MGXS):
-    """A fission multi-group cross section."""
+    # This is an abstract class which cannot be instantiated
+    __metaclass__ = abc.ABCMeta
```

Comment:
I'd agree -- no need to make things complicated. That being the case, the comment about not being able to instantiate should probably be removed.